src/data/automation_i18n.ts

```diff
@@ -513,36 +513,107 @@ export const describeCondition = (
   // State Condition
   if (condition.condition === "state") {
     let base = "Confirm";
-    const stateObj = hass.states[condition.entity_id];
-    const entity = stateObj
-      ? computeStateName(stateObj)
-      : condition.entity_id
-      ? condition.entity_id
-      : "an entity";
+    if (!condition.entity_id) {
+      return `${base} state`;
+    }
 
-    if ("attribute" in condition) {
-      base += ` ${condition.attribute} from`;
+    if (condition.attribute) {
+      const stateObj = Array.isArray(condition.entity_id)
+        ? hass.states[condition.entity_id[0]]
+        : hass.states[condition.entity_id];
+      base += ` ${computeAttributeNameDisplay(
+        hass.localize,
+        stateObj,
+        hass.entities,
+        condition.attribute
+      )} of`;
     }
 
-    let states = "";
+    let entities = "";
+    if (Array.isArray(condition.entity_id)) {
+      for (const [index, entity] of condition.entity_id.entries()) {
+        if (condition[entity]) {
+          entities += `${index > 0 ? "," : ""} ${
+            condition.entity_id.length > 1 &&
+            index === condition.entity_id.length - 1
+              ? condition.match === "all"
+                ? "and"
+                : "or"
+              : ""
+          } ${computeStateName(hass.states[entity]) || entity}`;
+        }
+      }
+    } else if (condition.entity_id) {
+      entities = hass.states[condition.entity_id]
+        ? computeStateName(hass.states[condition.entity_id])
+        : condition.entity_id;
+    }
 
+    if (!entities) {
+      // no entity_id or empty array
+      entities = "an entity";
+    }
+
+    base += ` ${entities} is`;
```

Comment:
```suggestion
    base += ` ${entities} is`;
```
Should we use `are` if these are multiple entities ?

Reply:
![image](https://user-images.githubusercontent.com/195327/228268187-e2cbb961-0a98-4351-ab12-ef4a90725802.png)
![image](https://user-images.githubusercontent.com/195327/228268265-207c819f-6a21-4def-85ec-464a3d74b5fa.png)